ivy/functional/ivy/nest.py

```diff
@@ -1015,6 +1015,49 @@ def nested_map(
         x following the applicable of fn to it's nested leaves, or x itself if x is not
         nested.
 
+    Examples
+    --------
+    With :code:`ivy.Array` inputs:
+
+    >>> def special_square(nested : float) -> float : return np.square(x)
+    >>> nested = ivy.Array([[1,2,3],[4,5]])
+    >>> results = ivy.nested_map(x = nested,
+    ...                   fn = special_square)
+    >>> print(results)
+    [[1, 4, 9], [16, 25]]
```

Comment:
Given your function input `nested` is an array, the function `nested_map` should return an array `array([[1,4, ....)` not a list

Reply:
It does return an array, but it is printed the same way as a list is. Should I change it to not be printing, but just returning?